crates/core/src/rpc/minimal.rs

```diff
@@ -752,6 +780,100 @@ impl Minimal for SurfpoolMinimalRpc {
 mod tests {
     use super::*;
     use crate::tests::helpers::TestSetup;
+    use solana_commitment_config::{CommitmentConfig, CommitmentLevel};
+    use solana_client::rpc_config::RpcContextConfig;
+    use solana_epoch_info::EpochInfo;
+
+    #[test]
+    fn test_get_block_height_basic() {
+        let setup = TestSetup::new(SurfpoolMinimalRpc);
+        
+        let result = setup.rpc.get_block_height(Some(setup.context.clone()), None);
+        assert!(result.is_ok(), "get_block_height should not fail");
+
+        let block_height = result.unwrap();
+
+        println!("Block height returned: {:?}", block_height);
+    }
```

Comment:
```suggestion
```
This test seems unnecessary - the `test_get_block_height_matches_svm_state` is doing the same test and more